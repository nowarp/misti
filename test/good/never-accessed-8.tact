contract TestContract {
    receive() {
        let test: Int = 123; // should be reported
    }
}
