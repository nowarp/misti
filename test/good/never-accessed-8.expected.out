test/good/never-accessed-8.tact:3:9:
  2 |     receive() {
> 3 |         let test: Int = 123; // should be reported
              ^
  4 |     }
Variable is never accessed
Help: Consider removing the variable
See: https://nowarp.github.io/tools/misti/docs/detectors/NeverAccessedVariables